src/components/FormModal/FormModal.js

```diff
@@ -4,12 +4,106 @@ import PropTypes from 'prop-types';
 
 import './FormModal.css';
 
-// import Search from '../../components/Search/Search';
+const FormModal = ({ user, toggleFormModal }) => {
+  const [title, setPostTitle] = useState('')
+  const [description, setPostDescription] = useState('')
+  const [image, setPostImageUrl] = useState('')
+
+  const submitPost = event => {
```

Comment:
This all looks good so far. Might not have ability to create a post in our MVP but this is definitely something to shoot for if we have time!